dev-infrastructure/templates/mgmt-cluster.bicep

```diff
@@ -8,7 +8,7 @@ param persist bool = false
 param currentUserId string
 
 @description('AKS cluster name')
-param aksClusterName string
+param aksClusterName string = 'aro-hcp-mgmt-aks1'
```

Comment:
why aks1?

Reply:
since the resource group already denotes the nature of the cluster, mgmt  or svc, just call it `aro-hcp-aks`?

Reply:
sorry it should be aro-hcp-mgmt-1. Currently we only have 1 aks cluster, but we would like to have the number at the end incase of future expansion (when we have more than 1 aks cluster)

Reply:
I love `aro-hcp-aks`

Reply:
i was wondering about the `1` suffix as well. i was under the impression we would expand MCs via additional resourcegroup, each containing a new MC, no?

Reply:
> I like aro-hcp-aks and we can use aro-classic-aks for new classic aks cluster

Not an issue for this thread, but to clarify, we won't have aro-classic-aks in the same resource group either

Reply:
thanks for the inputs, I have updated the naming in the shared [google doc](https://microsoft-my.sharepoint.com/:w:/p/gniranjan/ERWrqFehqZZDl3jRjaS0vrIB1MxGERKjhe-jByWIFT_PcA?e=Mb727L)